src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.String.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers.Binary;
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    internal ref partial struct CborValueReader
+    {
+        private static readonly System.Text.Encoding s_utf8Encoding = new System.Text.UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
+
+        // Implements major type 2 decoding per https://tools.ietf.org/html/rfc7049#section-2.1
+        public byte[] ReadByteString()
```

Comment:
@bartonjs how do you feel about this choice of naming? It's following naming convention from CBOR, but wondering whether we need to stick closer to .NET conventions (i.e. calling it `ReadByteArray()`)

Reply:
`ReadByteString` seems good enough for now.